Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -592,22 +589,22 @@ void MainWindow::ConnectGameList()
   connect(m_game_list, &GameList::OpenGeneralSettings, this, &MainWindow::ShowGeneralWindow);
 }
 
-void MainWindow::ConnectRenderWidget()
-{
-  m_rendering_to_main = false;
-  m_render_widget->hide();
-  connect(m_render_widget, &RenderWidget::Closed, this, &MainWindow::ForceStop);
-  connect(m_render_widget, &RenderWidget::FocusChanged, this, [this](bool focus) {
-    if (m_render_widget->isFullScreen())
-      SetFullScreenResolution(focus);
-  });
-}
-
 void MainWindow::ConnectHost()
 {
   connect(Host::GetInstance(), &Host::UpdateProgressDialog, this,
           &MainWindow::OnUpdateProgressDialog);
   connect(Host::GetInstance(), &Host::RequestStop, this, &MainWindow::RequestStop);
+  connect(Host::GetInstance(), &Host::UpdateRenderWidget, this,
```

Comment:
No need to pass `this` here.

Reply:
IIRC I had to pass a context parameter to be able to use `Qt::QueuedConnection`.

Reply:
Oh right, `Qt::QueuedConnection` is outside the lambda! nvm